src/components/dev-hub/audio.js

```diff
@@ -0,0 +1,256 @@
+    align-items: center;
+    display: grid;
+    column-gap: ${size.large};
+    justify-items: center;
+    grid-template-areas:
+        'image play slider badge options'
+        'details details details details details';
+    grid-template-columns: ${size.xlarge} ${size.large} auto 76px 88px;
+    grid-template-rows: ${ROW_SIZE}px auto;
+    max-width: ${size.maxWidth};
+    padding: 0 ${size.xlarge};
+    width: 100%;
+    @media ${screenSize.upToLarge} {
+        column-gap: ${size.xsmall};
+        grid-template-areas:
+            'play podcast-title close'
+            'slider slider slider'
+            'details details details'
+            '. expand .';
+        grid-template-columns: ${size.large} auto ${size.large};
+        grid-template-rows: ${ROW_SIZE}px auto auto 48px;
+        padding: 0 ${size.default};
+        row-gap: ${size.large};
+        ${({ isExpanded }) => !isExpanded && condensedMobilePlayerStyle};
+    }
+`;
+
+// Create grid-area containers for elements
+const CloseButton = styled(Button)`
+    grid-area: close;
+`;
+const ExpandButton = styled(Button)`
+    grid-area: expand;
+`;
+const ExpandedTitle = styled(H2)`
+    grid-area: exp-title;
+    @media ${screenSize.upToLarge} {
+        display: none;
+    }
+`;
+const ExpandedDescription = styled(P)`
+    color: ${colorMap.greyLightTwo};
+    grid-area: exp-desc;
+`;
+const PlayContainer = styled('div')`
+    grid-area: play;
+`;
+const PodcastTitleButton = styled(Button)`
+    font-family: akzidenz;
+    grid-area: podcast-title;
+    text-align: center;
+    text-decoration: none;
+    width: 100%;
+`;
+const StyledBadge = styled(Badge)`
+    grid-area: badge;
+    height: fit-content;
+    position: relative;
+`;
+const SliderContainer = styled('div')`
+    grid-area: slider;
+    width: 100%;
+`;
+const StyledImage = styled('img')`
+    border-radius: ${size.xsmall};
+    grid-area: image;
+`;
+
+const StyledExpandedContainer = styled('div')`
+    align-self: center;
+    background-color: ${colorMap.greyDarkThree};
+    column-gap: ${size.mediumLarge};
+    display: grid;
+    grid-area: details;
+    grid-template-areas: 'exp-title exp-desc exp-desc';
+    max-width: ${size.maxWidth};
+    padding: ${size.large} 0;
+    width: 100%;
+    @media ${screenSize.upToLarge} {
+        grid-template-areas: 'exp-desc';
+        padding: 0;
+    }
+`;
+
+const ExpandedContainer = ({ podcast }) => (
+    <StyledExpandedContainer>
+        <ExpandedTitle>{podcast.title}</ExpandedTitle>
+        <ExpandedDescription collapse>
+            {podcast.description}
+        </ExpandedDescription>
+    </StyledExpandedContainer>
+);
+
+const DesktopOptionsContainer = styled('div')`
+    display: flex;
+    grid-area: options;
+    justify-content: space-between;
+    width: 100%;
+    a {
+        padding-left: 0;
+        padding-right: 0;
+        width: ${size.large};
+    }
+`;
+
+const AudioPlayButton = React.memo(({ isPlaying, toggleIsPlaying }) => (
```

Comment:
Turns out the tab issues were that this components (and the options) kept being rerendered unnecessarily each second when hooks updated. I memoized these which prevented the extra rerenders and tabbing stays even after the slider updates.